### ISSUE TITLE
Fatal validator error if password.txt file is within same folder as keystores.

### ISSUE THREAD

**Issue by philknows** [ChainSafe/lodestar#4116]
**Describe the bug**

Currently, if a `password.txt` file is within the same folder as the keystores JSON within `./keystores`, there is a fatal error that persists.

```
Jun-04 23:08:02.054[]                 info: Lodestar network=prater, version=v0.38.0-dev.b5e24f7138, commit=b5e24f71389092c965e00b2c4c80c511d72fc4b1
 ✖ SyntaxError: Error parsing keystore at /keystores/password.txt: Unexpected token Y in JSON at position 0
    at JSON.parse (<anonymous>)
    at Function.parse (/usr/app/node_modules/@chainsafe/bls-keystore/lib/class.js:53:41)
    at file:///usr/app/node_modules/@chainsafe/lodestar-cli/src/cmds/validator/keys.ts:68:31
    at Array.map (<anonymous>)
    at getLocalSecretKeys (file:///usr/app/node_modules/@chainsafe/lodestar-cli/src/cmds/validator/keys.ts:63:21)
    at Object.validatorHandler [as handler] (file:///usr/app/node_modules/@chainsafe/lodestar-cli/src/cmds/validator/handler.ts:65:50)
```

**Expected behavior**

`password.txt` should be able to be read within the `./keystores` folder. Should not cause fatal error.

**Steps to Reproduce**

**Screenshots**
<img width="1063" alt="Screen Shot 2022-06-04 at 5 13 21 PM" src="https://user-images.githubusercontent.com/58080811/172028395-3b393b3f-a581-46b1-9c69-c388667b7db3.png">

<img width="743" alt="Screen Shot 2022-06-04 at 5 13 40 PM" src="https://user-images.githubusercontent.com/58080811/172028400-167e0a71-b0dd-4d48-a79a-d29c89472ab2.png">

**Comment by philknows** [comment]
Correct me if I'm wrong from a UX perspective... that this is not the default way that someone should setup the management of their keys? For #4112 under the key management section, I want to lay out the proper separation of how this should be setup, especially if `keystore.json` and `password.txt` should remain separate anyway for security reasons. 

Would `password.txt` be better placed in the `./secrets` folder instead?

**Comment by philknows** [comment]
From standup on Tuesday we decided that having keystores in the same directory as `password.txt` is not good practice. However, this bug is still valid as it shouldn't crash the validator client if it's done this way. We will re-adjust any setup guides and docs to ensure separation of the two.

**Comment by philknows** [comment]
Addressed by #4201
